devito/ir/iet/nodes.py

```diff
@@ -521,6 +534,31 @@ def write(self):
         return []
 
 
+class While(Node):
```

Comment:
So this is only a compiler level tool that is not accessible/usable at user level right?

Reply:
correct